src/modules/hacker/data/Phrases.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include "faker-cxx/Helper.h"
+#include "fmt/format.h"
+
+#include <string>
+#include <vector>
+
+#include "Abbreviations.h"
+#include "Ingverbs.h"
+#include "Nouns.h"
+#include "Verbs.h"
+#include "Adjectives.h"
+
+
+namespace faker {
+    // use fmt::format() to format each phrase w/ random word
+
+    const std::vector<std::string> phrases = {
+            fmt::format("If we {} the {}, we can get to the {} {} through the {} {} {}!", Helper::arrayElement<std::string>(verbs), Helper::arrayElement<std::string>(nouns), Helper::arrayElement<std::string>(abbreviations), Helper::arrayElement<std::string>(nouns), Helper::arrayElement<std::string>(adjectives), Helper::arrayElement<std::string>(abbreviations), Helper::arrayElement<std::string>(nouns)),
```

Comment:
we should not have generating in-place in data folder, consider leaving it as tokens {noun}, {verb} etc,  and matching it in phrase generation iterating through words and replacing corresponding tokens with generated values

Reply:
Yeah that was silly of me. I'll add tokens and select a random phrase in Hacker::Phrase and fill it in with random words and return it.